### DISCUSSION TITLE
wordpress导入无法直接自动生成摘要，或者能否添加一个总得开关，使得所有文章都能生成摘要

### DISCUSSION THREAD

**Discussion by king-zhangbn** [halo-dev/halo#6070] [Ideas]
### 你当前使用的版本

214

### 描述一下此特性

wordpress导入无法直接自动生成摘要，或者能否添加一个总得开关，使得所有文章都能生成摘要

**Comment by ruibaby** [comment]
建议向 https://github.com/halo-sigs/plugin-migrate 提交 issue。
